bigtable/snippets/filters/filters_test.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2020, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import uuid
+
+import datetime
+import pytest
+from google.cloud import bigtable
+
+from filter_snippets import filter_composing_chain, \
```

Comment:
Typically in python, with this import count, I would see `import filter_snippets` and then use as `filter_snippets.filter_composing_chain`. This isn't a hard and fast rule though. Is there a style preference for this in samples?

Reply:
I just went with this because I thought it was cleaner, I can change if you'd prefer